### PR TITLE
[WIP] 8200 graphical representation of the projects work package

### PR DESCRIPTION
**Pull request by sergiyvan**
The task is not finished yet, the PR is just to show how it could look.
Bower has downloaded the CSS of nvd3, but include doesn't work, so I did it manually.
screenshot from 2014-05-29 18 39 11

### CODE REVIEW

File: app/assets/stylesheets/default.css.sass [hunk 1]
```diff
@@ -64,3 +64,4 @@
 @import content/modal
 @import content/grid_table
 @import default/main
+@import nv.d3.min
```
Comment (myabc) [inline app/assets/stylesheets/default.css.sass:67]: @sergiyvan 

> Bower has downloaded the CSS of nvd3, but include doesn't work, so I did it manually.

Did you try using Sprocket's `//= require` directive?

File: app/assets/javascripts/project/show.js.erb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                  if(d.status_id<5){
+                    return "Open"; 
+                  }else{
+                    return "Closed";
+                  }
+            })
+            .key(function(d) { 
+              switch(d.type_id) {
+                case 1:
+                    return "Work Package"
```
Comment (myabc) [inline app/assets/javascripts/project/show.js.erb:15]: These shouldn't be hard-coded: the work package types are configurable, can vary per-project.

Comment (sergiyvan) [inline app/assets/javascripts/project/show.js.erb:15]: That why I written "The task is not finished yet, the PR is just to show how it could look." :)

**Comment by tessi** [comment]
FYI: I added the `[WIP]` 'tag' to your pull request title to show that it is <b>w</b>ork <b>i</b>n <b>p</b>rogress :)

You may remove that tag from the PR title once you think this PR is ready to be merged.

**Comment by lindenthal** [comment]
Closes because outdated

**Pull request closed without merging**